lib/api/uikit/class-beans-uikit.php

```diff
@@ -51,23 +61,44 @@ public function compile() {
 		 */
 		$styles = apply_filters( 'beans_uikit_euqueued_styles', $this->register_less_components() );
 
+		// If there are not styles to compile, bail out.
+		if ( empty( $styles ) ) {
+			return;
+		}
+
 		/**
-		 * Filter UIkit enqueued script components.
+		 * Filter UIkit style compiler arguments.
 		 *
 		 * @since 1.0.0
 		 *
-		 * @param array $components An array of UIkit script component files.
+		 * @param array $components An array of UIkit style compiler arguments.
 		 */
-		$scripts = apply_filters( 'beans_uikit_euqueued_scripts', $this->register_js_components() );
+		$args = apply_filters( 'beans_uikit_euqueued_styles_args', array() );
```

Comment:
`_enqueued_` or do we have to deprecate and then change the name?

Reply:
see the note below.